special-pages/pages/new-tab/app/activity/mocks/activity.mocks.js

```diff
@@ -40,6 +40,29 @@ export const activityMocks = {
             },
         ],
     },
+    longEntry: {
+        activity: [
+            {
+                favicon: { src: 'selco-icon.png' },
+                url: 'https://example.app',
+                title: 'example.app',
+                etldPlusOne: 'example.app',
+                favorite: false,
+                trackersFound: false,
+                trackingStatus: {
+                    trackerCompanies: [],
+                    totalCount: 0,
+                },
+                history: [
+                    {
+                        title: '/products/bathroom/toilets-and-bidets/wall-mounted/modern-collection/ceramic-toilet-bowl?color=white&size=standard&material=porcelain&inStock=true&freeShipping=true',
+                        url: 'https://example.com/products/bathroom',
+                        relativeTime: 'Just now',
+                    },
+                ],
+            },
+        ],
+    },
```

Comment:
This example was added to ensure the nested widgets still respect flex/grid things (eg: don't overflow)